### PR TITLE
Add checks in list wrapping that we don't wrap incomplete separated lists

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/63732

### CODE REVIEW

File: src/Features/Core/Portable/Wrapping/AbstractWrapper.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ protected AbstractSyntaxWrapper(IIndentationService indentationService)
             var sourceText = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
             foreach (var item in nodesAndTokens)
             {
-                if (item == null || item.Span.IsEmpty)
+                if (item == null || item.Span.IsEmpty || item.IsMissing)
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/Wrapping/AbstractWrapper.cs:50]: technically missing tokens can have non-empty spans (fi the compiler attached things like whitespace or other trivia to them).  Being more resilient to that as well.

File: src/EditorFeatures/CSharpTest/Wrapping/ParameterWrappingTests.cs [hunk 2]
```diff
@@ -984,6 +984,42 @@ public async Task TestWithMissingParameterList()
             await TestMissingAsync(
 @"class C {
     public void UpsertRecord<T>[||]
+}");
+        }
+
+        [Fact, WorkItem(61362, "https://github.com/dotnet/roslyn/issues/61362")]
```
Comment (ryzngard) [inline src/EditorFeatures/CSharpTest/Wrapping/ParameterWrappingTests.cs:990]: This ID is different than the bug listed in the summary

**Review by ryzngard — APPROVED** [review]
LGTM :shipit:

File: src/EditorFeatures/CSharpTest/Wrapping/ParameterWrappingTests.cs (hunk 2 above)
Comment (CyrusNajmabadi) [reply src/EditorFeatures/CSharpTest/Wrapping/ParameterWrappingTests.cs:990]: whoops.  will fix.

**Pull request merged**